low-level/actuators/impl/generic/realstor_encl.py

```diff
@@ -93,16 +93,16 @@ def perform_request(self, jsonMsg):
             ctrl_action = ""
             ctrl_type = ""
             if enclosure_request_data[-1] == "shutdown":
-                # "ENCL: enclosure:fru:controller:shutdown"
+                # "ENCL: enclosure:hw:controller:shutdown"
                 (request_type, _, component, component_type,
                     ctrl_action) = enclosure_request_data
             elif enclosure_request_data[-1] == "restart":
-                # "ENCL: enclosure:fru:controller:sc:restart"
-                # "ENCL: enclosure:fru:controller:mc:restart"
+                # "ENCL: enclosure:hw:controller:sc:restart"

```

Comment:
Make sure we communicate this to HA team as they plan to consume enclosure shutdown request for Feature F-8E, and also update the ticket with update request/response which introduced this.

Reply:
> no unknown value we should keep, default should be made cold_swabble where no backend tells whether its hot swappable, including solution specific config.

So for fan-module, Power supply & sideplane FRUs, we should return default cold_swappable str?

Reply:
done. Keeping the 'cold_swappable' as default unit.

>And please confirm we have tickets for HA & CSM to consume these changes, as applicable & capture in jira also

yes.. created tickets for same.